applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -753,25 +759,22 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
                                                                                     point_number,
                                                                                     this->GetIntegrationMethod());
 
-                double integration_weight = GetIntegrationWeight(integration_points,
+                integration_weight = GetIntegrationWeight(integration_points,
                                                                     point_number,
                                                                     this_kinematic_variables.detJ0);
 
-                if (dimension == 2 && this->GetProperties().Has(THICKNESS))
-                    integration_weight *= this->GetProperties()[THICKNESS];
+                if (dimension == 2 && r_properties.Has(THICKNESS))
```

Comment:
why do you think avoiding the GetX... is faster? it should be inlines and be exactly the same...

Reply:
Ok! I thought that maybe it was better. In any case it looked cleaner right? :)